examples/contrib/cifar10/main.py

```diff
@@ -76,16 +76,21 @@ def training(local_rank, config):
     trainer = create_trainer(model, optimizer, criterion, lr_scheduler, train_loader.sampler, config, logger)
```

Comment:
I hope you tried to run the example, because I think it won't work because of the `train_step` function, the return of `train_step` is a dict, and that's not supported for now with the LRFinder, the output must be of type float, and we are extending that in this PR by supporting tensors #2045 , so this must be edited https://github.com/pytorch/ignite/blob/ba8c63c358c96a5bdce428b708e7934c8e543a65/examples/contrib/cifar10/main.py#L329 
Check this method for more details https://github.com/pytorch/ignite/blob/master/ignite/handlers/lr_finder.py#L137

Reply:
@KickItLikeShika I think we can use `output_transform` to transform dict to float, see https://github.com/pytorch/ignite/blob/05f2b558f6e10bd8b05b6157684c392e65473638/ignite/handlers/lr_finder.py#L370
We can keep the output as it is now and add `output_transform` to `lr_finder.attach`

Reply:
Nice idea, thanks @vfdev-5

Reply:
Great, thanks for the input. Should I add the `output_transform` in the `utils.py` or in `main` only?

Reply:
I have added it in `main.py` for now, let me know if it has to be moved.